CITATION.cff

```diff
@@ -0,0 +1,40 @@
+cff-version: 1.2.0
+title: "OCCA: A portable and vendor-neutral framework for parallel programming on heterogeneous platforms"
+message: "Please cite the following works when using this software."
+authors:
+  - family-names: "Medina"
+    given-names: "David S"
+  - family-names: "St-Cyr"
+    given-names: "Amik"
+  - family-names: "Warburton"
+    given-names: "Tim"
+    orcid: "https://orcid.org/0000-0002-3202-1151"
```

Comment:
I think we should put the OCCA developers under authors here, not the authors of the
preferred citation. See [here](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#authors).